docs/buttons.md

```diff
@@ -30,7 +30,7 @@ You can find them in two sizes: the default `.btn` and the smaller `.btn-sm`.
 
 ## Primary
 
-Primary buttons are green and are used to indicate the *primary* action on a page. When you need your shit to stand out, rock the `.btn.btn-primary`. You can use it on both button sizes—just add `.btn-primary`.
+Primary buttons are green and are used to indicate the *primary* action on a page. When you need your the buttons stand out, use the `.btn.btn-primary`. You can use it with both button sizes—just add `.btn-primary`.
```

Comment:
Nice change! :tada:

Though, I think "your buttons" reads better than "your the buttons;" unless you intended "you are the buttons."